### PR TITLE
updated how to cryptoparty section

### PR DESCRIPTION
**Pull request by dawningsuncryptoparty**
The "How To CryptoParty" had been updated on cryptoparty.in months ago. The handbook now has the same version.

### CODE REVIEW

File: src/chapter_00_introducing_cryptoparty/how_to_cryptoparty.md [hunk 1]
```diff
@@ -1,84 +1,306 @@
...
+## Introduction
 
- * Make sure you have Internet connectivity and enough power sources for all devices. If you do not have a place to hold a CryptoParty, find a pub or park where you can meet and squeeze the public bandwidth. That will really hone your skills!
+CryptoParty is a global and decentralized grass-roots movement. As such, they vary as much as they are adapted to their specific environments. This How To is written to give you some ideas on what might work and what not so much, but as with all direct action: plans are nothing, planning is everything, and it is all good as long as you follow CryptoParty's [guiding principles](https://cryptoparty.in/guiding_principles): be excellent to each other and do things.
 
- * Bring USB sticks and printed handouts for those who need them, and set up old computers for people to fiddle with and try out new skills.
+As time has shown that there is some general tactics which make everything go much more easy, this is our attempt in writing it down (please contribute - see “Edit this page” in the upper left corner).
```
Comment (yuvadm) [inline src/chapter_00_introducing_cryptoparty/how_to_cryptoparty.md]: Not everyone will read this online, maybe the last part is irrelevant in some cases?

File: src/chapter_00_introducing_cryptoparty/how_to_cryptoparty.md [hunk 2]
```diff
@@ -1,84 +1,306 @@
...
+This list is only a suggestion. Learning groups form around these topics and depending on how large the space is, how many people attend to learn and how many are willing to conduct a learning group they can be grouped more broadly or specific.
+
+What you offer depends on the angels available. Further suggestions are listed in a separate [tools overview](https://cryptoparty.in/learn/tools). All of them are free and open source software. And of course we like decentralized services.
+
+* discussion and orientation table
+* encrypt emails with [PGP](https://en.wikipedia.org/wiki/Pretty_Good_Privacy)
+* encrypt your IMs with [XMPP](https://en.wikipedia.org/wiki/XMPP) and [OTR](https://en.wikipedia.org/wiki/Off-the-Record_Messaging)
+* anonymous surfing [Tor](https://torproject.org)
+* privacy enhancing browser plug-ins
+* mobile security ([Android](https://cryptoparty.in/learn/tools#android), [iOS][https://cryptoparty.in/learn/tools#ios)
```
Comment (yuvadm) [inline src/chapter_00_introducing_cryptoparty/how_to_cryptoparty.md]: Syntax error here

**Comment by yuvadm** [comment]
Looks good! Thanks for the work @dawningsuncryptoparty :)

Any other comments?

**Comment by dawningsuncryptoparty** [comment]
Thanks for the feedback and incorporated them into the pull request.

**Comment by dawningsuncryptoparty** [comment]
I would also suggest a new order of the elements in the "introducing CryptoParty" folder.

This is the current order....

* about this book
* how to CryptoParty
* party like it's dec 31st 1983
* preface
* why privacy matters

And here's my proposal...

* about this book
* preface
* why privacy matters
* CryptoParty manifesto
* party like it's dec 31st 1983
* how to CryptoParty

**Comment by dawningsuncryptoparty** [comment]
i changed the sub chapter's order as described above, too....going to merge it now.

**Pull request merged**